### PR TITLE
CON-1634 update corewar subject

### PR DESCRIPTION
**Pull request by nprimo**
Make subject consistent with new VM version

### CODE REVIEW

File: subjects/corewar/README.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ The players will be able to create new processes using specific instructions, it
 
 There is a special instruction, called `live` that notifies the VM one player is still healthy and doing well.
 
-Every once in a while the VM will check if the players executed such instruction and, if they didn't, all the processes created by this player will be instantly killed.
+Every once in a while the VM will check which processes executed such instruction and kill all the processes that have not.
+After this check, the VM will consider that every remaining process is dead until it successfully executes a `live` instruction.
```
Comment (mikysett) [inline subjects/corewar/README.md:48]: This line may be confusing... Considering a process to be "dead" may means this process should not perform any action, maybe we can rephrase it to better explain the expected behavior.

**Review by mikysett — CHANGES_REQUESTED** [review]
The section `#### Stop player execution` at line `123` is not valid anymore and should be modified according the new rules.

Maybe it could be nice to change the title also into `#### Stop process execution`

File: subjects/corewar/README.md (hunk 1 above)
Comment (nprimo) [reply subjects/corewar/README.md]: I rephrased it
